### ISSUE TITLE
Correctly parse boolean attribute values

### ISSUE THREAD

**Issue by Silind** [Silind-Software/direflow#125]
## Feature

Correctly parse boolean attribute values

#### Currently
```html
// 'disabled' evaluates to false
<web-component disabled ></web-component>
```
#### Desired
The value `disabled` should be considered true.

### Currently
```html
// 'disabled' evaluates to true
<web-component disabled="false"></web-component>
```
#### Desired
The value `disabled` should be considered a boolean, and evaluate to false

#### :information_source:  Update
According to the HTML standard, values of `true` and `false` are not supported.
https://stackoverflow.com/questions/4139786/what-does-it-mean-in-html-5-when-an-attribute-is-a-boolean-attribute

As @Silentscripter scripter points out below, boolean values in attributes should be limited to the presence or absence of the attribute rather than it's value.

**Comment by Silentscripter** [comment]
I'm ready to handle this.
Are we sure to allow disabled="false" as falsy conversion? Shouldn't false values be limited only to not adding the attribute itself (as it happens with standard html tags attributes?)

**Comment by fabiofranzini** [comment]
Hi, there are update on this issue?

**Comment by Silind** [comment]
Moving this back to todo, so this one is again up for grabs 😄

**Comment by johnmcase** [comment]
Should this be closed now that it was released in 3.5?

**Comment by Silind** [comment]
Ahh, thank you!
Yes, this needs to be closed now :smile:
